### PR TITLE
feat(docs): Leave Policy Assignment

### PR DESCRIPTION
**Pull request by Anurag810**
image

### CODE REVIEW

File: erpnext_documentation/www/docs/user/manual/en/human-resources/hr-settings.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ An email is sent to all the employees of the company when an employee has a birt
 ### 1.4 Expense Approver Mandatory In Expense Claim
 In Expense Claim Document the 'Expense Approver' field is set to mandatory on checking this option.
 
+> Payroll Settings will be part of HR Settings till version 12. In version 13, Payroll Settings will be part of our new Module Payroll.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/hr-settings.md:33]:
```suggestion
> Payroll Settings will be part of HR Settings till version 12. In version 13, Payroll Settings will be part of the new module, Payroll.
```

File: erpnext_documentation/www/docs/user/manual/en/human-resources/hr-settings.md [hunk 2]
```diff
@@ -90,6 +92,11 @@ If checked, the system generates a draft Leave Encashment record on the expiry o
 ### 3.6 Restrict Backdated Leave Application
 If checked, the system will not allow making a backdated leave application.
 
+> Introduced in version 13
+
+### 3.7 Automatic Allocate Leaves Based On Leave Policy
+If checked, The leave will be granted to employee automatically on Effective From Date as per present Leave Policy Assignment.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/hr-settings.md:98]:
```suggestion
If checked, leaves will be granted to the employees automatically based on the Effective From date as per the present Leave Policy Assignment.
```

File: erpnext_documentation/www/docs/user/manual/en/human-resources/leave-period.md [hunk 3]
```diff
@@ -30,7 +30,11 @@ Additionally, you can check the 'Is Active' checkbox if you want to enable this
 <img class="screenshot" alt="Leave Period"
 	src="{{docs_base_url}}/assets/img/human-resources/leave-period.png">
 
-Once the information is saved, the Leave Period will also be used as a tool to help you grant leaves for a category of employees. 
+## 3. Granting leave using Leave period
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/leave-period.md:33]:
```suggestion
## 3. Granting leave using Leave Period
```

File: erpnext_documentation/www/docs/user/manual/en/human-resources/leave-period.md [hunk 4]
```diff
@@ -30,7 +30,11 @@ Additionally, you can check the 'Is Active' checkbox if you want to enable this
...
+
+> It was the part of ERPNext till version 12 now, from version 13 leave will be granted using [Leave Policy Assignment](/docs/user/manual/en/human-resources/leave-policy-assignment).
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/leave-period.md:35]:
```suggestion
> In version 12, leaves could be granted via Leave Period through the "Grant Leaves" button. In version 13, leaves will be granted using [Leave Policy Assignment](/docs/user/manual/en/human-resources/leave-policy-assignment).
```

File: erpnext_documentation/www/docs/user/manual/en/human-resources/leave-period.md [hunk 5]
```diff
@@ -30,7 +30,11 @@ Additionally, you can check the 'Is Active' checkbox if you want to enable this
...
+
+Once the information is saved, the Leave Period will also be used as a tool to help you grant leaves for a category of employees.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/leave-period.md:37]: is this also in v13? @Anurag810

File: erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
+# Leave Policy Assignment
+
+> Introduced in Version 13
+
+Leave Policy Assignment in ERPNext is used to grant leaves to employees. To access Leave policy assignment, go to:
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md:5]: is this to grant leaves or to assign a leave policy to an employee?

File: erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+Before creating a Leave Policy Assignment, it is advisable to create the following:
+
+* [Employee](/docs/user/manual/en/human-resources/employee)
+* [Leave Policy](/docs/user/manual/en/human-resources/leave-policy)
+
+## 2. How to create a Leave Policy Assignment
+
+1. Go to Leave Policy Assignment, click on New.
+1. Select Employee and Leave Policy.
+1. Select Assignment based on if required.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md:20]:
```suggestion
1. Select Assignment based on the following as needed:
```

File: erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+    * If an assignment is based on Leave period, You need to select Leave period. The Effective From and Effective To will be set automatically.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md:21]:
```suggestion
    * If "Assignment based on" is set to Leave Period, you need to select the applicable Leave Period. The Effective From and Effective To dates will be set automatically based on the Leave Period selected.
```

File: erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+    * If an assignment is based on Joining Date, The Effective From date will be set to Employee's Date of Joining.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md:22]:
```suggestion
    * If "Assignment based on" is set to Joining Date, the Effective From date will be set to the employee's Date of Joining.
```

File: erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+    * If an assignment is based on is blank, You need to set Effective From and Effective To Manually.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md:23]:
```suggestion
    * If "Assignment based on" is left blank, then you will have to set the Effective From and Effective To date manually.
```

File: erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md [hunk 11]
```diff
@@ -0,0 +1,74 @@
...
+1. Save and Submit.
+
+<img class="screenshot" alt="Leave Policy Assignment"
+	src="{{docs_base_url}}/assets/img/human-resources/leave-policy-assignment.png">
+
+## 3. Granting Leave
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md:29]:
```suggestion
## 3. Granting Leaves
```

File: erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md [hunk 12]
```diff
@@ -0,0 +1,74 @@
...
+
+Once the information is saved, the Leave Policy Assignment will also be used as a tool to help you grant leaves for employees. The **Grant Leave** button will appear at the right top corner.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md:31]:
```suggestion
Once the information is saved, the Leave Policy Assignment will also be used as a tool to help you grant leaves to employees. The **Grant Leave** button will appear at the right top corner.
```

File: erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md [hunk 13]
```diff
@@ -0,0 +1,74 @@
...
+
+
+<img class="screenshot" alt="Grant Leaves"
+	src="{{docs_base_url}}/assets/img/human-resources/leave-policy-assignment-grant-leave.png">
+
+On click, The Leave allocation will be created automatically based on [Leave Policy](/docs/user/manual/en/human-resources/leave-policy) as shown below.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md:37]:
```suggestion
On clicking the "Grant Leave" button, Leave Allocation will be automatically created based on the [Leave Policy](/docs/user/manual/en/human-resources/leave-policy) as shown below.
```

File: erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md [hunk 14]
```diff
@@ -0,0 +1,74 @@
...
+
+<img class="screenshot" alt="Leave Allocations"
+	src="{{docs_base_url}}/assets/img/human-resources/granted-leaves.png">
+
+>**Note:** The Grant Leave button will appear only then if, No leaves are granted or there is no allocation against current Leave Policy Assignment.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md:42]:
```suggestion
>**Note:** The Grant Leave button will appear only if there are no leaves granted or there is no allocation against a current Leave Policy Assignment.
```

File: erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md [hunk 15]
```diff
@@ -0,0 +1,74 @@
...
+
+## 4. Automatic Allocate Leaves Based On Leave Policy
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md:44]:
```suggestion
## 4. Automatically Allocate Leaves Based On Leave Policy
```

File: erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md [hunk 16]
```diff
@@ -0,0 +1,74 @@
...
+
+To enable Automatic allocation for your Leave Policy Assignment you need to Enable [Automatic Allocate Leaves Based On Leave Policy ](/docs/user/manual/en/human-resources/hr-settings#37-automatic-allocate-leaves-based-on-leave-policy) checkbox in HR Settings. Our Scheduler Run background Job and check the Effective From Date in Leave Policy Assignment and create Leave allocation automatically.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md:46]:
```suggestion
To enable automatic allocation of leaves based on the Leave Policy Assignment, enable [Automatic Allocate Leaves Based On Leave Policy ](/docs/user/manual/en/human-resources/hr-settings#37-automatic-allocate-leaves-based-on-leave-policy) checkbox in HR Settings. The scheduler then runs a background job and checks the Effective From date in the Leave Policy Assignment to create leave allocations automatically.
```

Comment (michellealva) [reply erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md:46]: @Anurag810 change the checkbox name to "Automatically Allocate Leaves Based On Leave Policy"

File: erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md [hunk 17]
```diff
@@ -0,0 +1,74 @@
...
+
+>**Note:** The Leave will be automatically granted only if, No leave is granted against that Leave Policy Assignment till Effective From date.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md:48]:
```suggestion
>**Note:** Leaves will be automatically granted only if, there are no leaves granted against that Leave Policy Assignment till Effective From date.
```

Comment (michellealva) [reply erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md:48]: till effective from date or to date? @Anurag810

File: erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md [hunk 18]
```diff
@@ -0,0 +1,74 @@
...
+
+## 5. Features
+### 5.1 Bulk Leave Policy Assignment
+
+Our System also allow to create multiple Leave Policy Assignment for multiple employees.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md:53]:
```suggestion
You can also bulk assign the Leave Policy to multiple employees in a single click.
```

File: erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md [hunk 19]
```diff
@@ -0,0 +1,74 @@
...
+
+1. Go to Leave Policy Assignment list, click on Bulk Leave Policy Assignment.
+1. Dialog Will appear, Select Employee. You can filter Employee based on Company and Department or You can also use standard filters on clicking Add Filters.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md:56]:
```suggestion
1. A dialog will appear. Select the Employee. You can filter employees based on Company and Department or you can also use standard filters by clicking the Add Filters button.
```

File: erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md [hunk 20]
```diff
@@ -0,0 +1,74 @@
...
+1. Select Leave Policy and Effective From and Effective To.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md:57]:
```suggestion
1. Select Leave Policy and Effective From and Effective To dates.
```

File: erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md [hunk 21]
```diff
@@ -0,0 +1,74 @@
...
+1. Click on Assign.
+
+<img class="screenshot" alt="Leave Allocations"
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md:60]:
```suggestion
<img class="screenshot" alt="Bulk Leave Policy Assignment" src="{{docs_base_url}}/assets/img/human-resources/bulk-leave-policy-assignment.png">
```

File: erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md [hunk 22]
```diff
@@ -0,0 +1,74 @@
...
+	src="{{docs_base_url}}/assets/img/human-resources/bulk-leave-policy-assignment.png">
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md:61]:
```suggestion
	
```

File: erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md [hunk 23]
```diff
@@ -0,0 +1,74 @@
...
+
+### 5.2 Granting leaves to multiple employees
+
+Our System also allow to Grant Leaves based on multiple Leave Policy Assignment for multiple employees.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md:65]:
```suggestion
Leaves can be granted based on multiple Leave Policy Assignments to multiple employees.
```

File: erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md [hunk 24]
```diff
@@ -0,0 +1,74 @@
...
+
+1. Go to Leave Policy Assignment list, click on Grant Leaves.
+1. Dialog Will appear, Select Leave Policy Assignments. You can filter Assignment based on Company and Employee or You can also use standard filters on clicking Add Filters.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md:68]:
```suggestion
1. A dialog will be shown. Select the Leave Policy Assignments. You can filter the assignments based on Company and Employee or you can also use standard filters by clicking on the Add Filters button.
```

File: erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md [hunk 25]
```diff
@@ -0,0 +1,74 @@
...
+1. Select Leave Policy Assignment.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md:69]:
```suggestion
1. Select the Leave Policy Assignments.
```

**Review by michellealva — CHANGES_REQUESTED** [review]

File: erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md [hunk 26]
```diff
@@ -0,0 +1,73 @@
+# Leave Policy Assignment
+
+> Introduced in Version 13
+
+Leave Policy Assignment in ERPNext is used to grant leaves to employees. To access Leave policy assignment, go to:
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md:5]: @Anurag810 add more info here, how will the user know what this feature does and why was it created?

Comment (Anurag810) [reply erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md:5]: not required.

**Pull request merged**